### PR TITLE
Add Inherit case

### PR DESCRIPTION
**Pull request by candyleer**
Please fill up those following information before send Pull request 

* Pull request NO: <The pull request no that your pull to skywalking>
* Agent repository URL: <Your skywalking repository git url>
* Testcases repository URL: <Your skywalking-agent-testcases repository git url>

### CODE REVIEW

File: spring-scenario/config/expectedData.yaml [hunk 1]
```diff
@@ -3,15 +3,45 @@ registryItems:
...
+      spanType: Entry
+      peer: ''
+      peerId: 0
+      tags:
+      - {key: url, value: 'http://inherit-scenario:8080/inherit-case/inherit/child/test'}
+      - {key: http.method, value: GET}
+      refs:
+      - {parentServiceId: 0, parentServiceName: /resttemplate-case/case/resttemplate,
+         networkAddressId: 0, entryServiceId: 0, refType: CrossProcess, parentSpanId: 6,
+         parentTraceSegmentId: not null, parentApplicationInstanceId: nq 0,
```
Comment (ascrutae) [inline spring-scenario/config/expectedData.yaml]: I suggest you should assert the `parentTraceSegmentId`.  the format of parentTraceSegmentId looks like this: `${Application[index]}`. Here is the  [doc](https://github.com/SkywalkingTest/skywalking-agent-testcases/blob/master/docs/how-to-write-a-plugin-testcase-cn.md#%E6%9C%9F%E6%9C%9B%E6%95%B0%E6%8D%AE%E6%96%87%E4%BB%B6)

File: spring-scenario/spring-projectA/src/main/java/test/apache/skywalking/apm/testcase/resttemplate/controller/Config.java [hunk 2]
```diff
@@ -10,9 +10,11 @@
     private static Logger logger = LogManager.getLogger(Config.class);
     private static String projectBURL;
     private static String projectCURL;
+    private static String projectDURL;
 
     static {
-        InputStream inputStream = Config.class.getClassLoader().getResourceAsStream("/spring-config.properties");
+        InputStream inputStream = Config.class.getClassLoader().getResourceAsStream("spring-config.properties");
+//        InputStream inputStream = Thread.currentThread().getContextClassLoader().getResourceAsStream("spring-config.properties");
```
Comment (ascrutae) [inline spring-scenario/spring-projectA/src/main/java/test/apache/skywalking/apm/testcase/resttemplate/controller/Config.java]: I suggest this code should be remove.

**Comment by candyleer** [comment]
fixed above issue

**Comment by wu-sheng** [comment]
@ascrutae Let's run all again? and make the PR merged.

**Comment by ascrutae** [comment]
Got it. The test case was running

**Review by wu-sheng — APPROVED** [review]

**Pull request merged**